pkg/apis/kubevirt/v1/types.go

```diff
@@ -71,6 +71,9 @@ type VersionSpec struct {
 
 type ComponentSpec struct {
 	Version string `json:"version,omitempty"`
+	Affinity v1.Affinity `json:"affinity,omitempty"`
```

Comment:
Please add comment descriptions above struct fields in order to not break the OpenAPI schema of the CRD.
See here for examples: https://github.com/kubevirt/kubevirt-ssp-operator/pull/225/commits/d6e77a2a07ecc0cc0ef36837c4390c9c186909f6#diff-5d05db759e7d0cd7129d4d2f1229d7afR127

Reply:
Done